source/common/local_reply/local_reply.cc

```diff
@@ -23,7 +23,7 @@ class BodyFormatter {
         content_type_(Http::Headers::get().ContentTypeValues.Text) {}
 
   BodyFormatter(const envoy::config::core::v3::SubstitutionFormatString& config,
-                Server::GenericFactoryContextImpl context)
+                Server::Configuration::GenericFactoryContext& context)
```

Comment:
optional: think it's worth doing a typedef for this one just to avoid the churn here and for folks downstream?